scipy/integrate/_bvp.py

```diff
@@ -810,6 +810,10 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
         mesh interval ``norm(r / (1 + abs(f)) < tol``, where ``norm`` is
         estimated in a root mean squared sense (using a numerical quadrature
         formula). Default is 1e-3.
+    tol_bc : float, optional
+        Desired tolerance applied to boundary conditions. Boundary condition 
```

Comment:
I suggest just stating that this is a desired absolute tolerance. Something like 
```
Desired absolute tolerance for the boundary condition residuals: `bc` value
should satisfy ``abs(bc) < tol_bc`` component-wise. Equals to `tol` by default.

Reply:
A slightly modified version of  this text was added to the docstring.